docs/source/checks/simple.rst

```diff
@@ -0,0 +1,175 @@
+.. _simple:
+
+Simple Check
+====================
+
+Description
+-----------
+
+The SimpleCheck gives you basic information about your datasets and detects the most obvious of datashifts.
+
+Some shifts result in the change of basic metrics, that are commonly used for the analysis and exploration of datasets.
+Those metrics and statistics are calculated on both datasets and compared afterwards. HIER NOCH AMAZON DEEQU ERWÄHNEN,
+WIRD NACH UMBENENNUNG DES CHECKS GEMACHT
+
+The basic check should be used in most analyses because its metrics are scientifically grounded and commmonly used.
```

Comment:
I don't really like this sentence at all. The other checks are scientifically grounded too. The argumentation that you should use it, because it's commonly used isn't good too...

Reply:
doubled to the comment above? If not Im not sure to which line you are referring to in the comment above. 

Thats not true. Things that are commonly used in science are a good indicator for a solid metric. That has nothing to do with the check being so great, it goes more in the direction that metrics like that are standard metrics to describe data in datascience. Which is a good indicator of the metric being useful.